xcixor/iReporter
# Bug missing body parameters 162336915

## What is the Purpose of this PR
The purpose of this PR is add a means to verify that all body parameters are present
## Why is it important
It prevents the app from crashing if the user does not provided all the parameters
## Tasks completed
- Add required parameter to parser for required fields
## How can it be tested 
After cloning the repo test an endpoint with the headers set to application/json
make a request and omit some of the body parameters, it should notify you of the missing fields
## Relevant PT story
[#162336915](https://www.pivotaltracker.com/story/show/162336915)
